crates/astria-sequencer-utils/src/config.rs

```diff
@@ -0,0 +1,17 @@
+use clap::Parser;
+
+#[derive(Debug, Parser)]
+pub struct Config {
+    #[clap(long)]
+    pub source_genesis_file: String,
```

Comment:
same for these. If you have `/// <what does this opt do>` above the field, clap will give you a description.